packages/thumbprint-react/components/Modal/index.tsx

```diff
@@ -384,17 +367,17 @@ class Modal extends React.Component {
         }
     }
 
-    render() {
+    render(): JSX.Element {
         const {
             children,
-            isOpen,
+            isOpen = false,
             onCloseClick,
             onCloseFinish,
             onOpenFinish,
-            shouldCloseOnCurtainClick,
-            shouldHideCloseButton,
-            width,
-            heightAboveSmall,
+            shouldCloseOnCurtainClick = true,
+            shouldHideCloseButton = false,
+            width = 'medium',
+            heightAboveSmall = 'auto',
         } = this.props;
```

Comment:
Why are the default values needed here?

Reply:
why wouldn't they be? these are the values that were previously provided by `defaultProps`

Reply:
My bad. I got confused because `ModalAnimatedWrapper` and `Modal` have a lot of overlap. This is correct though.